sql-schema/131.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `config` (`config_name`, `config_value`, `config_default`, `config_descr`, `config_group`, `config_group_order`, `config_sub_group`, `config_sub_group_order`, `config_hidden`, `config_disabled`) VALUES('webui.old_availability_map', 'false', 'false', 'Availability map old view', 'webui', 0, 'graph', 0, '1', '0');
```

Comment:
You've already added this in sql-schema/129.sql

Once that's removed then we're good for a merge.